src/js/index.d.ts

```diff
@@ -27,15 +27,21 @@ declare class UACustomEvent {
   ): void;
 }
 
-declare interface Operation {
+declare interface TagGroupOperation {
```

Comment:
This could technically be interpreted as an interface change, but it's just a typescript definition for object parameters, and it's also for the module, not for public use. I don't think it's a problem.